pom.xml

```diff
@@ -150,6 +156,14 @@
 					<artifactId>maven-openmrs-plugin</artifactId>
 					<version>1.0.1</version>
 				</plugin>
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-surefire-plugin</artifactId>
+					<version>2.22.1</version>
+					<configuration>
+						<argLine>-Xmx512m -Djdk.net.URLClassPath.disableClassPathURLCheck=true</argLine>
```

Comment:
Do we need this argLine?

Reply:
no, we don't need this argLine, I added this to fix the testNG suit issue, but the issue is still there.